devices/HumanWrenchProvider/HumanWrenchProvider.cpp

```diff
@@ -845,19 +1264,23 @@ bool HumanWrenchProvider::attach(yarp::dev::PolyDriver* poly)
     // Get the device name from the driver
     const std::string deviceName = poly->getValue("device").asString();
 
-    if (deviceName == "human_state_provider") {
+    yInfo()<<LogPrefix<<"Attaching "<< deviceName;
+
+    if (deviceName == "human_state_provider" || deviceName == "human_state_remapper" ) {
 
         // Attach IHumanState interface from HumanStateProvider
         if (pImpl->iHumanState || !poly->view(pImpl->iHumanState) || !pImpl->iHumanState) {
             yError() << LogPrefix << "Failed to view IHumanState interface from the polydriver";
             return false;
         }
 
-        // Check the interface
-        if (pImpl->iHumanState->getNumberOfJoints() == 0
+        //TODO Check the interface
+        int count = 1000000;
```

Comment:
As before, couldn't we just use a timeout (also based on your experience on the reason why this was happening) ?

Reply:
As per https://github.com/robotology/human-dynamics-estimation/pull/309#discussion_r960750199.